app/javascript/components/search/controls/download/DownloadButton.js

```diff
@@ -5,13 +5,15 @@ import Tooltip from 'react-bootstrap/lib/Tooltip'
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger'
 
 import DownloadSelectionModal from './DownloadSelectionModal'
-import { hasSearchParams } from 'providers/StudySearchProvider'
 import { UserContext } from 'providers/UserProvider'
 
 /**
- * Component for "Download" button and Bulk Download modal.
+ * Component for "Download" button which shows a Bulk Download modal on click.
+ * DemoMode is included which will limit the number of TDR files queried, and make sure
+ * analysis files are seeded as well.  Ordinarily, we wouldn't merge demo code, but this feature may be in demo-only
+  * state for a long time, and need lots of demos
  */
-export default function DownloadButton({ searchResults={} }) {
+export default function DownloadButton({ searchResults={}, isDemoMode=false }) {
```

Comment:
If `isDemoMode` is only used for determining which files to show I wonder if it would make sense to call it something more file specific like `shouldShowDemoFiles` ?

Reply:
I like the idea, but `shouldShowDemoAppropriateFiles` seems overly long.  Maybe `showDemoFiles`?

Reply:
ah yes `showDemoFiles` is much better 👍